.jules/friction/open/config_edge_cases.md

```
@@ -0,0 +1,11 @@
+# Friction Item: Config Edge Cases
+
+## Observation
```

markdownlint-cli2 (0.22.1)

[warning] Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)